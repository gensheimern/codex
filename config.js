module.exports = {
	DB_HOST: process.env.DB_HOST || '159.65.115.221',
	DB_USER: process.env.DB_USER || 'root',
	DB_PASS: process.env.DB_PASS || 'teamcodex',
	DB_NAME: process.env.DB_NAME || 'lunch_planner',
	JWT_SECRET: process.env.JWT_SECRET || 'secret',
	MAIL_HOST: process.env.MAIL_HOST || '159.65.115.221',
	MAIL_USER: process.env.MAIL_USER || 'support@codex-team.de',
	MAIL_PASS: process.env.MAIL_PASS || 'teamcodex123',
};
