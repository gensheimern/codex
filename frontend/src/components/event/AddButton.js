import React from 'react';
import { Link } from 'react-router-dom';

export default class AddButton extends React.Component {

	render() {
		return (
			<Link to="/newEvent">
				<div style={{
					width: "30px",
					height: "30px",
					borderRadius: "50%",
					backgroundColor: "#aaa",
					float: "left",
					marginLeft: "2%",
					fontSize: "18pt",
				}}>&nbsp;&nbsp;+</div>
				<h2 style={{color: "white",}}>&nbsp;ADD EVENT</h2>
			</Link>
		);
	}

}