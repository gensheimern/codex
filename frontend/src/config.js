const config = {
	debug: process.env.REACT_APP_DEBUG === 'TRUE',
	apiPath: process.env.REACT_APP_API_PATH,
	wsPath: process.env.REACT_APP_WS_PATH,
	basePath: process.env.REACT_APP_BASE_PATH,
}

export default config;