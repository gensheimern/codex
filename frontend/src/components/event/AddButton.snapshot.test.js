/**
 * @author:Stella Neser
 *  Komponente ist ein Snapshottest
 */
import React from 'react';
import renderer from'react-test-renderer';
import AddButton from './AddButton';
import config from '../../config';
import {MemoryRouter} from 'react-router-dom';

const fetchMock = require('fetch-mock');

describe('AddButton Snapshot', () => {
    test('renders', () =>{
        const component = renderer.create(
            <MemoryRouter>
                <AddButton/>
            </MemoryRouter>
        );
        let tree= component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});