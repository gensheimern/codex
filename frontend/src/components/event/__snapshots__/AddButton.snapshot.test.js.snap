// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton Snapshot renders 1`] = `
<a
  href="/newEvent"
  onClick={[Function]}
>
  <div
    style={
      Object {
        "backgroundColor": "#aaa",
        "borderRadius": "50%",
        "float": "left",
        "fontSize": "18pt",
        "height": "30px",
        "marginLeft": "2%",
        "width": "30px",
      }
    }
  >
      +
  </div>
</a>
`;
